server/utils/dateHelpers.ts

```diff
@@ -127,7 +127,7 @@ export const formatDateTime = (
 
 export const timeFormat = (dateTimeStr: string) => {
   const date = new Date(dateTimeStr)
-  const hours = date.getUTCHours()
-  const minutes = date.getUTCMinutes() === 0 ? '00' : date.getUTCMinutes()
+  const hours = date.getHours() < 10 ? `0${date.getHours()}` : date.getHours()
+  const minutes = date.getMinutes() === 0 ? '00' : date.getMinutes()
```

Comment:
You also need to account for minutes 1 to 9 having a leading 0
Rather than doing these manual conversions, could just replace the whole method content with:
```
return new Date(dateTimeStr).toLocaleTimeString('en-gb', { hour: '2-digit', minute: '2-digit' })
```